string command
string request_key
string request_value
---
bool result
string result_value
